### PR TITLE
Redo Section 1 Tweak Section 0

### PR DESCRIPTION
**Pull request by kingthorin**
This PR fixes OWASP/wstg#344.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

* The following sections were removed from section 1 (they don't need to be maintained by the WSTG team and only add page count to the TG):
  - board members
  - employees and contractors
  - the OWASP foundation
  - OWASP foundation bylaws
  - core values
  - core purpose
  - code of ethics
  - principles
  - project
  - privacy policy
  - membership or donations
  - licensing (already covered in 1_Frontispiece.md)

* The following sections were integrated into 1_Frontispiece.md:
  - Contacting OWASP
  - Follow Us

* Section 0 was revised as follows
  * Foreword by Eoin Keary
    - Dropped board reference and outdated link in heading.
    - Removed references to projects which are dead or haven't been
  migrated to the new OWASP site.
  * The Role of Automated Tools
    - Removed old link.
  * Call to Action
    - Updated membership link.
    - Removed sign-off link.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Will fix the lint issues in about an hour.

File: document/1_Frontispiece/1_Frontispiece.md [hunk 1]
```diff
@@ -54,3 +54,12 @@ This document is released under the [Creative Commons 4.0 License](https://creat
 - Visa is a registered trademark of VISA USA.
 
 All other products and company names may be trademarks of their respective owners. Use of a term in this document should not be regarded as affecting the validity of any trademark or service mark.
+
+## Contacting OWASP
+
+Contact details for the [OWASP Foundation](https://owasp.org/) are available [online](https://owasp.org/contact/). If you have a question concerning a particular project, we strongly recommend using the [mailing list](https://groups.google.com/a/owasp.org/forum/) for that project. Many questions can also be answered by searching the [OWASP](https://owasp.org/) web site, so please check there first.
```
Comment (victoriadrake) [inline document/1_Frontispiece/1_Frontispiece.md]: Are we still calling it a “mailing list”?

Comment (kingthorin) [reply document/1_Frontispiece/1_Frontispiece.md]: I'll tweak it to sound less 1996

**Comment by kingthorin** [comment]
Done

**Pull request merged**